xcube_stac/store.py

```diff
@@ -249,10 +249,10 @@ def _is_valid_data_type(cls, data_type: DataTypeLike) -> bool:
         by the store.
 
         Args:
-            data_type: Data type that is to be checked.
+            data_type (DataTypeLike): Data type that is to be checked.
```

Comment:
```suggestion
            data_type: Data type that is to be checked.
```

Redundant. Types added by API doc generators like mkdocs.